batchflow/models/tf/encoder_decoder.py

```diff
@@ -411,13 +411,14 @@ def decoder(cls, inputs, name='decoder', **kwargs):
                         elif letter in ['u']:
                             if upsample.get('layout') is not None:
                                 x = cls.upsample(x, name='upsample', **upsample_args)
+                        elif letter == 'c':
```

Comment:
Maybe add `s` (stands for `skip`)?

Reply:
I think it will be misleading. We are combining inputs here, not adding skips.